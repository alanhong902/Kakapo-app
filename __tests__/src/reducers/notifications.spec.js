import { prop } from 'ramda';
import configureStore from 'stores/configureStore';
import { notifyActions } from 'actions/';

const store = configureStore();

const currState = () => prop('notifications', store.getState());

test('[reducer/notifications]', () => {
  store.dispatch(notifyActions.notify('testID', 'test'));
  expect(currState()).toMatchSnapshot();

  store.dispatch(notifyActions.clear('testID'));
  expect(currState()).toMatchSnapshot();
});
