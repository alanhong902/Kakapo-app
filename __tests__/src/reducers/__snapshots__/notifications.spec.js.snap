// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[reducer/notifications] 1`] = `
Object {
  "shortIDGeneratedStr": undefined,
  "testID": "test",
}
`;

exports[`[reducer/notifications] 2`] = `
Object {
  "shortIDGeneratedStr": undefined,
}
`;
